### DISCUSSION TITLE
obs config option for pointstat

### DISCUSSION THREAD

**Discussion by ghost** [dtcenter/METplus#2178] [Configuration] [answered]
Hi,
I am referring to an earlier discussion in here to sort out configuration option for OBS
https://github.com/dtcenter/METplus/discussions/1888#discussioncomment-4383151

Based on the suggestion here I put something like this 
OBS_VAR1_NAME = obs_val
OBS_VAR1_LEVELS = Z550
OBS_VAR1_OPTIONS = obs_index1 = [ 550 ]; obs_index2 = [ AOD ]

But it results 0 matched pairs. I need your suggestion to rightly point to the verifying field in netcdf file. 
Script on Hera is /scratch1/NCEPDEV/stmp2/Partha.Bhattacharjee/METplus/Scripts/Aeronet_EVS/Scripts/Pointstat_AeronetAOD2.conf
asciitonc are in /scratch1/NCEPDEV/stmp2/Partha.Bhattacharjee/METplus/Scripts/Aeronet_EVS/Asciitonc

Any suggestions ?
Thanks

**Comment by jprestop** [comment, marked as answer]
Hi @perthsb.

Looking at the output of a run:

> DEBUG 2: Observations processed    = 683165
DEBUG 2: Rejected: station id      = 0
DEBUG 2: Rejected: obs var name    = 683165

I can see that all of the observations were rejected due to the obs var name.  Looking at your Pointstat_AeronetAOD2.conf, I see that you have:

> OBS_VAR1_NAME = AOD

but then later you reset OBS_VAR1_NAME:

> OBS_VAR1_NAME = obs_val

If I change the second instance of OBS_VAR1_NAME to "AOD" instead of obs_val, I get some matched pairs:

> DEBUG 3: Number of matched pairs   = 198
DEBUG 3: Observations processed    = 683165
DEBUG 3: Rejected: station id      = 0
DEBUG 3: Rejected: obs var name    = 378200
DEBUG 3: Rejected: valid time      = 299847
DEBUG 3: Rejected: bad obs value   = 0
DEBUG 3: Rejected: off the grid    = 0
DEBUG 3: Rejected: topography      = 0
DEBUG 3: Rejected: level mismatch  = 4920

Please reduce your OBS section definitions to only one section to eliminate redundancy and confusion.  Also, one last thing.  I noticed that you are pointing to a very old version of the PointStatConfig_wrapped file:

> POINT_STAT_CONFIG_FILE = /scratch1/BMC/dtc/Julie.Prestopnik/METplus/METplus-3.0/parm/met_config/PointStatConfig_wrapped

I suggest updating that to be:

> POINT_STAT_CONFIG_FILE = /contrib/METplus/METplus-5.0.1/parm/met_config/PointStatConfig_wrapped

Please let us know how it goes and if you have any other questions.

**Reply by ghost** [reply]
Thanks @jprestop. I overlooked the repeat part of the obs declaration in the config file. It is working fine now. Thanks
